cmd/bridge/main.go

```diff
@@ -203,7 +203,7 @@ func startRelay(ctx *cli.Context, version string) error {
 		return err
 	}
 
-	messenger, err := buildNetMessenger(cfg, marshaller)
+	messenger, err := buildNetMessenger(cfg, marshaller, log)
```

Comment:
should use a different logger instance, perhaps `p2p`, in order to differentiate them